src/main/java/eu/europa/ted/efx/model/CallStackObject.java

```diff
@@ -6,6 +6,6 @@
  * As the EfxExpressionTranslator translates EFX to a target language, the objects in the call-stack
  * are typically code snippets in the target language.
  */
-public abstract class CallStackObject {
+public interface CallStackObject {
```

Comment:
I renamed it to ParsedEntity since these "objects" are actually outputs of the various parser methods, pushed to the stack for subsequent methods to pop and use as inputs.